ccdaservice/ccda_gateway.php

```diff
@@ -75,7 +75,7 @@
 if (!$parameterArray ['view']) {
     header('Content-Type: application/xml');
 } else {
-    $h = '<a href="./../portal/home.php" </a><button style="color: red; background: white;" >' . xlt("Return Home") . '</button><br>';
+    $h = '<a href="./../portal/home.php" </a><button style="color: red; background: white;" >' . xlt("Return Home") . '</button><br />';
```

Comment:
Testing: If this is connected to themes it has to be changed to the classes `text-danger` and `bg-white`